docs/deployment/linux/red-hat.txt

```diff
@@ -0,0 +1,141 @@
+.. _red-hat:
+
+============================
+Run CrateDB on Red Hat Linux
+============================
+
+CrateDB maintains the official RPM repositories for:
+
+- `Red Hat Linux 7`_
+- `Red Hat Linux 6`_
+
+.. _Red Hat Linux 7: https://www.redhat.com/en/resources/whats-new-red-hat-enterprise-linux-7
+.. _Red Hat Linux 6: https://www.redhat.com/en/resources/red-hat-enterprise-linux-6
+
+Both of these work with RedHat Enterprise Linux, CentOS, and Scientific Linux.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Configure YUM
+=============
+
+All CrateDB packages are signed with GPG.
+
+To get started, you must import the CrateDB public key, like so:
+
+.. code-block:: sh
+
+   sh$ sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
+
+You must then install the CrateDB repository definition.
+
+For Red Hat Linux 7, run:
+
+.. code-block:: sh
+
+   sh$ sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/noarch/crate-release-7.0-1.noarch.rpm
+
+For Red Hat Linux 6, run:
+
+.. code-block:: sh
+
+   sh$ sudo rpm -Uvh https://cdn.crate.io/downloads/yum/6/x86_64/crate-release-6.5-1.noarch.rpm
+
+The above command creates two new repository configuration files:
```

Comment:
I've been looking at the `crate-release-7.0-1.noarch.rpm` package again, and what it actually does is installing a single file called `crate.repo` in `/etc/yum.repos.d/`
the file contains both the stable and testing repo information, where testing repo is disabled

```
[crate]
name=CRATE Packages for Enterprise Linux 7
baseurl=https://cdn.crate.io/downloads/yum/7/noarch
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-crate

[crate-testing]
name=CRATE Packages for Enterprise Linux 7 - Testing
baseurl=https://cdn.crate.io/downloads/yum/testing/7/noarch
enabled=0
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-crate
```